### ISSUE TITLE
deploy in real server

### ISSUE THREAD

**Issue by anggorodhanumurti** [kuliks08/mock-api-server#1]
can this code deploy in a real server i have debian server i want to test it there not in localhost

**Comment by kuliks08** [comment]
Of course, you can deploy it on any server or hosting that supports PHP. Just place all the files in the root folder.
